src/lib/button/button.scss

```diff
@@ -136,6 +136,7 @@
 .mat-button, .mat-flat-button, .mat-stroked-button, .mat-raised-button, .mat-icon-button,
 .mat-fab, .mat-mini-fab {
   .mat-button-wrapper > * {
+    pointer-events: none;
```

Comment:
Since we project user content into the wrapper, doesn't this have the potential of breaking existing behavior if somebody was binding events to it?

Reply:
I think that's true. It's technically a breaking change. In general, I think it makes sense to enforce consistent behavior though. So, probably just wait till V8? 

Also a small note in the `button.md` might be cool too. (optional though)

Reply:
They wouldn't have access to binding to events on the wrapper since its in the `MatButton`'s template though right?

Reply:
Not on the wrapper itself, but on the things that are projected inside the wrapper. `pointer-events` propagates to all the children.